run_route_scripts/run_with_server_matrix.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
```

Comment:
do we actually need a separate script for this, the only thing that should really differ is the parsing of the response? the other script i think can let you run any file of requests so it shoudl work for the input part. and the output can be handled pretty easily by simply looking at the url to find which endpoint was hit route/sources_to_targets whatever and calling appropriate handler to parse the resulting json

Reply:
i dont feel strongly about it though, but it feels to me like tech debt

Reply:
tbh, the vast majority of python in this repo feels like tech debt (except for most installable scripts)😄 I should’ve done the work to clean this all up a bit more and not just copy paste. I’ll do that at some point I’m sure. If you want I can keep this locally instead of committing until I find the motivation to do that:)

Reply:
nah its ok, i trust you to keep it on your list :smile: